tests/test_gym_envs.py

```diff
@@ -194,3 +194,28 @@ def test_render(env_config):
         ), f"Expected 4 channels in the rendered image, got {frame.shape[-1]}."
 
     env.close()
+
+@pytest.mark.parametrize("env_config", _ALL_ENV_CONFIGS)
+def test_human_render(env_config):
```

Comment:
Instead of a whole new function, I think what we can do is modify the above function to have the following signature:

```python
@pytest.mark.parametrize("env_config", _ALL_ENV_CONFIGS)
@pytest.mark.parametrize("render_mode", ["human", "rgb_array"])
def test_render(env_config, render_mode):
    """Test that pyflyt rendering works."""
    env = gym.make(
        env_config[0],
        render_mode=render_mode,
        **env_config[1],
    )
    ...
```